tests/aws/services/events/conftest.py

```diff
@@ -66,6 +66,21 @@ def _create_event_bus(**kwargs):
             LOG.warning(f"Failed to delete event bus {event_bus_name}: {e}")
 
 
+@pytest.fixture
+def events_create_default_or_custom_event_bus(events_create_event_bus, region_name, account_id):
+    def _create_default_or_custom_event_bus(event_bus_type: str = "default"):
+        if event_bus_type == "default":
+            event_bus_name = "default"
+            event_bus_arn = f"arn:aws:events:{region_name}:{account_id}:event-bus/default"
+        if event_bus_type == "custom":
```

Comment:
elif

Reply:
I used else since any bus type that is not default should create a custom event bus